Tasks/DotNetCoreCLIV2/task.json

```
@@ -191,7 +191,7 @@
         "vstsFeed"
       ],
       "type": "pickList",
-      "label": "Use packages from this Azure Artifacts feed",
+      "label": "Use packages from this Azure Artifacts feed. Please just enter the feed name, not an entire url",
       "defaultValue": "",
       "helpMarkDown": "Include the selected feed in the generated NuGet.config. You must have Azure Artifacts installed and licensed to select a feed here.",
```

This isn't part of your change, but while you're here:
```suggestion
      "helpMarkDown": "Include the selected feed in the generated NuGet.config.\n\nFormat:\n* For organization-scoped feeds: `FeedName`\n* For project-scoped feeds: `ProjectName/FeedName`",
```
* The deleted part is no longer correct, and hasn't been in ages -- artifacts is included (up to 2GB) with Azure DevOps
* We can be a bit more verbose here than in the label

---

Changed this!